src/Symfony/Component/Intl/CONTRIBUTING.md

```diff
@@ -1,91 +0,0 @@
-Contributing to the Intl component
```

Comment:
this is very outdated AFAIK (cc @jakzal)

Reply:
Indeed :)